### DISCUSSION TITLE
Realy in Andes For IEEE14

### DISCUSSION THREAD

**Discussion by Adarsh-365** [CURENT/andes#490] [Q&A]
I am working on the IEEE14 model want add industrial Realy(model) in it How can achieve that plz help 🙏

**Comment by jinningwang** [comment]
For now, we have Distributed Generation protection model, https://docs.andes.app/en/latest/groupdoc/DGProtection.html#dgprotection. Other relay models can be achieved using ``Toggle``.

**Comment by Adarsh-365** [comment]
Could we schedule a video call(Metting) to discuss

**Reply by jinningwang** [reply]
A rough idea for your reference: using pert file to do the condition check at every step and adjust the control parameters if triggered.
Here is a discussion of how to use pert file, https://github.com/CURENT/andes/discussions/471
